### PR TITLE
Added docker compose config for running zally with github integration

### PR DESCRIPTION
**Pull request by vitold**
Fixes https://github.com/zalando-incubator/zally/issues/532

### CODE REVIEW

**Comment by vitold** [comment]
👍

**Comment by maxim-tschumak** [comment]
@vitold Feel free to merge this to `docker-compose.yaml` - it's meant to run the whole stack (all Zally components)

**Comment by vitold** [comment]
@maxim-tschumak github integration config requires two environment variables - GITHUB_OAUTH_TOKEN and GITHUB_SECRET so if you don't have it (don't need github integration) you will see stack traces in logs.

**Comment by ruiaraujo** [comment]
@vitold Then I propose another solution, let's only have this compose file have the bark service which is an optional add-on.

Then when people want to start everything, they can run:
```docker-compose -f docker-compose.yaml -f bark-service.yaml up```

which yields the same result.

**Comment by ruiaraujo** [comment]
:+1:

**Comment by maxim-tschumak** [comment]
Great idea @ruiaraujo ! 
@vitold Please also adjust the README file (`Quick start guide` section)

File: build_and_run.sh [hunk 1]
```diff
@@ -16,5 +16,16 @@ cd ${SERVER_DIR} && ./gradlew clean build
...
 cd ${WEB_UI_DIR} && npm install && npm build
 
-# Docker-compose
-cd ${SCRIPT_DIR} && docker-compose up --build
+if [ "$@" == "--bark" ]; then
+
+    GITHUB_INTEGRATION_DIR=${SCRIPT_DIR}/github_integration
+
+    #Build Bark
+    cd {GITHUB_INTEGRATION_DIR} && ./gradlew clean build
```
Comment (maxim-tschumak) [inline build_and_run.sh]: `${GITHUB_INTEGRATION_DIR}` - `$` sign is missing

**Comment by vitold** [comment]
CDP/pr/lifecycle — Lifecycle failed. This amazing CDP check is failing without any particular reason. No logs, no results, just failure. Can someone help to tame this creature?

**Comment by maxim-tschumak** [comment]
@vitold weird, the last PR533 build was actually successful (see dev console). Please merge master into your branch - that will trigger another (clean) build.

File: README.md [hunk 2]
```diff
@@ -47,6 +47,16 @@ executing this script:
 
 Web UI is accessible on `http://localhost:8080`; Zally server on `http://localhost:8000`
 
+*To run zally with github integration*:
+```bash
+export GITHUB_OAUTH_TOKEN=your_github_oauth_token
+export GITHUB_SECRET=your_github_secret
+
+./build_and_run.sh --bark
```
Comment (maxim-tschumak) [inline README.md]: -> `./build-and-run.sh`

File: README.md [hunk 3]
```diff
@@ -47,6 +47,16 @@ executing this script:
...
+``` 
+Github webhook endpoint is acceible on http://localhost:8081/github_webhook
```
Comment (maxim-tschumak) [inline README.md]: `acceible` -> `accessible`

**Comment by maxim-tschumak** [comment]
:+1:

**Comment by netme** [comment]
👍

**Comment by netme** [comment]
@maxim-tschumak could you please re-approve?

**Comment by maxim-tschumak** [comment]
:+1:

**Pull request merged**